debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresValueConverter.java

```diff
@@ -676,7 +676,9 @@ protected Object convertBits(Column column, Field fieldDefn, Object data, int nu
                     bitset.set(len - i - 1);
                 }
             }
-            data = bitset;
+            int numBits = bitset.length();
```

Comment:
Can you add a comment here on the purpose of this?

Reply:
here's the gist of the PR!

Reply:
instead of passing numBytes which is length/precision of BITVARYING divided by 8 bits/byte downstream we only use the length of the input. the source datastore normally does the "heavy lifting" for us already.
I will add a comment here on that in a bit.

Reply:
Ok, makes sense. My only question is why this wasn't a problem before? Perhaps the PG driver update changed the behavior?

Reply:
Maybe, I don't think such essential part was recently changed. We didn't have any test for that type yet. I think the only thing changed was that we now have users with that field data type? I need to test with an old version.

Reply:
> I need to test with an old version.

Yes, please look at the Jira issue, they state there the previous version that seemed to work.

Reply:
I'm currently merging locally tests for BIT VARYING to 1.0 branch in order to check it. WIP